### DISCUSSION TITLE
RLS on StripeWrapper FDW

### DISCUSSION THREAD

**Discussion by Witteborn** [supabase/supabase#28726] [Questions] [answered]
I could not activate RLS on the tables from the StripeWrapper. In the TableEditor it says that RLS is disabled on foreign tables. I tried enabling them through the SQL editor but it didn't work.

Any ideas on how I can limit access on my foreign tables to only admin access?

**Comment by GaryAustin1** [comment, marked as answer]
https://supabase.com/docs/guides/database/extensions/wrappers/overview#security
